_data/software/freecad.yml

```diff
@@ -0,0 +1,38 @@
+---
+schemaVersion: '1.0'
+description:
+  whatItDoes:
+    en: >-
+      FreeCAD is a general-purpose parametric 3D CAD modeler and a building
+      information modeling (BIM) software with finite-element-method (FEM)
+      support.
+    fr: >-
+      FreeCAD est un modeleur CAO 3D paramétrique à usage général et un logiciel
+      de modélisation des informations du bâtiment (BIM) avec prise en charge de
+      la méthode des éléments finis (FEM).
+category: '43232600'
+homepageURL:
+  en: 'https://www.freecadweb.org/'
+  fr: 'https://www.freecadweb.org/?lang=fr'
+licences:
+  - URL:
+      en: 'https://github.com/FreeCAD/FreeCAD/blob/master/LICENSE'
+      fr: 'https://github.com/FreeCAD/FreeCAD/blob/master/LICENSE'
+    spdxID: GPL-2.0-or-later
```

Comment:
Not a big one, but I couldn't find the "or later" part. And oddly found here talking about LGPL https://www.freecadweb.org/wiki/About_FreeCAD which doesn't really make sense for a GUI app